WordPress/src/main/res/layout-land/jetpack_powered_bottom_sheet.xml

```diff
@@ -18,59 +16,79 @@
-        android:text="@string/wp_jetpack_get_new_jetpack_app"
-        android:textAppearance="?attr/textAppearanceSubtitle2"
-        android:textColor="?attr/colorOnSecondary"
-        app:layout_constraintTop_toBottomOf="@id/caption"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"/>
+                <com.airbnb.lottie.LottieAnimationView
+                    android:id="@+id/illustration_view"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/margin_extra_extra_medium_large"
+                    android:scaleType="centerCrop"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:lottie_rawRes="@raw/wp2jp_left"
+                    app:lottie_enableMergePathsForKitKatAndAbove="true"
+                    app:lottie_autoPlay="true"
+                    app:lottie_loop="false" />
+
+                <TextView
+                    android:id="@+id/title"
+                    style="@style/TextAppearance.Material3.HeadlineMedium"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/margin_extra_medium_large"
+                    android:text="@string/wp_jetpack_powered_better_with_jetpack"
+                    android:textFontWeight="700"
+                    app:layout_constraintTop_toBottomOf="@id/illustration_view"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    tools:targetApi="p" />
+
+                <TextView
+                    android:id="@+id/caption"
+                    style="?attr/textAppearanceBody1"
+                    android:text="@string/wp_jetpack_powered_features"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/margin_extra_medium_large"
+                    app:layout_constraintTop_toBottomOf="@id/title"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"/>
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </ScrollView>
 
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/primary_button"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_extra_extra_medium_large"
+            android:layout_marginEnd="@dimen/margin_extra_extra_medium_large"
+            android:layout_marginBottom="@dimen/margin_64dp"
```

Comment:
Should this bottom margin be reduced in the landscape mode?

Reply:
@

Reply:
> Should this bottom margin be reduced in the landscape mode?

It's a good idea @ravishanker!
I replaced the `64dp` bottom margin with `32dp` 🙇 in commit https://github.com/wordpress-mobile/WordPress-Android/pull/17503/commits/8ff221676b628abd5ea73b523709f1aef8891edc

> Does this issue exist on full screen bottom sheet layouts?

It does, though afaik there's still work-in-progress on the full screen overlays in this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/17432, and my understanding is that it will replace the full screen bottom sheet layouts that were prepared in `JetpackPoweredBottomSheetFragment`.

Reply:
Resolving this since it was addressed on the other overlays as well: https://github.com/wordpress-mobile/WordPress-Android/pull/17540#issuecomment-1327438837